org.sugarlabs.MusicBlocks.json

```diff
@@ -10,6 +10,7 @@
         "--socket=fallback-x11",
         "--socket=pulseaudio",
         "--device=dri",
+        "--filesystem=xdg-download",
```

Comment:
What I think we need is display a proper file chooser, instead of that very unintuitive dialog.